components/tikv_util/src/logger/mod.rs

```diff
@@ -31,7 +31,7 @@ pub const DATETIME_ROTATE_SUFFIX: &str = "%Y-%m-%d-%H:%M:%S%.f";
 const SLOG_CHANNEL_SIZE: usize = 10240;
 // Default is DropAndReport.
 // It is not desirable to have dropped logs in our use case.
-const SLOG_CHANNEL_OVERFLOW_STRATEGY: OverflowStrategy = OverflowStrategy::Block;
+const SLOG_CHANNEL_OVERFLOW_STRATEGY: OverflowStrategy = OverflowStrategy::Drop;
```

Comment:
Comments need to be updated.

Reply:
Would `DropAndReport` be better?

Reply:
@L3T please take a look at comment. You can file a new PR to address comment. And make sure all the cherrypick PRs have comments updated, thanks.

Reply:
> Would `DropAndReport` be better?

Good question, we kind of discussed this strategy as well. The agreement was overflowing message isn't much useful to both users and developers therefore `Drop` is an appropriate choice. However it is definitely open to discussion before our long term solution is implemented.

Reply:
Thanks for your reply.

A drop report log can inform us the logger is busy in a cheap way. Although we are working on letting this information be shown on grafana, I feel better with an error can bring many different forms of warnings.

Reply:
DropAndReport inform us by a line of log message which is for human being only. We can neither know anything about it from metrics nor the context of dropped log was about. And that's why we thought it was not useful at the moment and used Drop instead.